Documentation/FrameworkOverview.md

```diff
@@ -8,36 +8,33 @@ learning about new modules and finding more specific documentation.
 For examples and help understanding how to use RAC, see the [README][] or
 the [Design Guidelines][].
 
-## Streams
+## Signals
+
+A **signal**, represented by the [Signal][] class, is any series of objects.
 
-A **stream**, represented by the [RACStream][] abstract class, is any series of
-object values.
+<!-- TODO: This is something I don't know, is it still true for Signal?  -->
+<!-- Signals are [monads][]. Among other things, this allows complex operations to be
+built on a few basic primitives (`-bind:` in particular). [Signal][] also
+implements the equivalent of the [Monoid][] and [MonadZip][] typeclasses from
+[Haskell][]. -->
```

Comment:
We should delete this. This may help a small number of people that already know FP but has a lot more potential to confuse newcomers that don't. If we want to talk about this it should be a buried a lot deeper in the docs.